server/id/id.go

```diff
@@ -155,6 +179,8 @@ func (alloc *allocatorImpl) rebaseLocked() error {
 	alloc.metrics.idGauge.Set(float64(end))
 	alloc.end = end
 	alloc.base = end - alloc.step
+	log.Info("rebase id allocator", zap.String("label", alloc.label),
```

Comment:
Do we need to check `checkCurrEnd` before printing the log?

Reply:
print it too. https://github.com/tikv/pd/pull/5181/commits/0f0a83347c4b6ad6eb8c7552c0e9197cbeca840a

Reply:
see https://github.com/tikv/pd/pull/5181/files#diff-f7546c173fbab9027155ba08485b80bc7d570970a5e38d86a016d82cf7a6effeR181-R184